apps/auth/oauth2.py

```
@@ -27,19 +44,185 @@
+                    "status": "error",
+                    "error_id": error_id,
+                    "envy_error": "300x4",
+                    "type": "MissingEmailError",
+                    "status_code": 400,
+                }
+            )
+            raise HTTPException(
+                status_code=400,
+                detail=f"Email not provided by OAuth provider --ENVYSTART--ERROR:300x4;ERROR_ID:{error_id}--ENVYEND--"
+            )
+        username = user_info.get("name", email)
```

Ruff (0.8.2)

Local variable `username` is assigned to but never used

Remove assignment to unused variable `username`

(F841)